config/environments/deployed_development.rb

```diff
@@ -44,7 +44,8 @@
   config.active_storage.service = :amazon
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.ssl_options = { exclude: ->(request) { request.path.include?("healthcheck") } }
```

Comment:
As per https://github.com/DFE-Digital/early-careers-framework/pull/3588/files#r1262781810 change to "check" here